test/e2e/shared/aws.go

```diff
@@ -2366,3 +2366,39 @@ func GetMountTargetState(e2eCtx *E2EContext, mountTargetID string) (*string, err
 	}
 	return result.LifeCycleState, nil
 }
+
+// ///////////////////////////////////////////////////////////////////////////////
+func getAvailabilityZone() string {
+	return "us-east-1a"
+}
+
+func getInstanceType() string {
+	return "t3.large"
+}
+
+func GetDedicatedHost(e2eCtx *E2EContext) (string, error) {
```

Comment:
Since this is actually allocating, we might want to rename this function. Seeing `Get` makes me think it's just retrieving one that already exists.